docs/modules/panel_configuration/pages/expressions.adoc

```diff
@@ -0,0 +1,68 @@
+:imagesdir: ../assets/images
+[.lead]
+[[pc-expressions]]
+= Expressions
+
+{page-component-title} supports xref:https://commons.apache.org/proper/commons-jexl/index.html [JEXL] mathematical and conditional operators in performance data queries.
+Use them to combine or otherwise transform one or more series into a new series.
+For example, you could specify a filter to display the SUM of data from different attribute queries (e.g., available memory on two nodes).
+
+This section provides sample expressions to illustrate how to use them.  
+For additional documentation on expressions, see the following:
+
+* https://docs.opennms.org/opennms/releases/26.1.0/guide-development/guide-development.html#_measurements_api[Measurements API documentation] of your OpenNMS version
```

Comment:
Links with specific version numbers can be tricky and get very quickly outdated and are extremely hard maintaining. We have an alias with `latest` instead of `26.1.0` in our docs to address this problem. It will float with the latest stable release. You can use this link here instead: https://docs.opennms.org/opennms/releases/latest/guide-development/guide-development.html

Reply:
> Links with specific version numbers can be tricky and get very quickly outdated and are extremely hard maintaining. We have an alias with `latest` instead of `26.1.0` in our docs to address this problem. It will float with the latest stable release. You can use this link here instead: https://docs.opennms.org/opennms/releases/latest/guide-development/guide-development.html

Doh! I will update that.

Reply:
It's still there https://github.com/OpenNMS/opennms-helm/commit/552d473ce31c207b8a5621e8b3cd1219b5ce7a4b#diff-62c92cb1f31e7eb7dfdc6300151b1bc7R13

Reply:
That has been updated in a later commit. #17 in 552d473 is the specific part of the commit to deal with HELM-231.

Reply:
Sorry. Didn't notice that.